lib/modules/manager/maven/update.ts

```diff
@@ -78,7 +79,24 @@ export function bumpPackageVersion(
     const startTagPosition = versionNode.startTagPosition;
     const versionPosition = content.indexOf(versionNode.val, startTagPosition);
 
-    const newPomVersion = semver.inc(currentValue, bumpVersion);
+    let newPomVersion: string | null = null;
+    const prerelease = semver.prerelease(currentValue);
```

Comment:
```suggestion
    const currentPreleaseValue = semver.prerelease(currentValue);
```

This renaming of variable will make things more readable